### PR TITLE
Update import components, add robots.txt, sitemap files

### PR DESCRIPTION
**Pull request by yurickv**

### CODE REVIEW

File: app/sitemap.xml [hunk 1]
```diff
@@ -0,0 +1,242 @@
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <url>
+    <loc>https://gym-adrenalin.com.ua</loc>
+    <lastmod>2024-01-01T15:02:24.021Z</lastmod>
```
Comment (vitaliy-kro) [inline app/sitemap.xml:4]: Do we need this?

File: app/sitemap.xml [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+    <changefreq>yearly</changefreq>
```
Comment (vitaliy-kro) [inline app/sitemap.xml:5]: change to always

File: app/sitemap.xml [hunk 3]
```diff
@@ -0,0 +1,242 @@
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
```
Comment (vitaliy-kro) [inline app/sitemap.xml:1]: You can dynamically generate it for every route, using js

File: app/layout.tsx [hunk 4]
```diff
@@ -1,10 +1,13 @@
 import './globals.css';
+import dynamic from 'next/dynamic';
 import { Inter } from 'next/font/google';
-import { Header } from '@/components/Header';
-import { Footer } from '@/components/Footer';
+import Header from '@/components/Header';
+// import Footer from '@/components/Footer';
```
Comment (vitaliy-kro) [inline app/layout.tsx:5]: delete unused code

File: components/main-page/GalerySection.tsx [hunk 5]
```diff
@@ -1,4 +1,9 @@
-import GalerySectionCard from './GalerySectionCard';
+import dynamic from 'next/dynamic';
+
+// import GalerySectionCard from './GalerySectionCard';
```
Comment (vitaliy-kro) [inline components/main-page/GalerySection.tsx:3]: unsused

**Review by vitaliy-kro — CHANGES_REQUESTED** [review]

File: app/sitemap.xml (hunk 3 above)
Comment (yurickv) [reply app/sitemap.xml:1]: how?

**Review by vitaliy-kro — APPROVED** [review]

**Pull request merged**